docs/modules/ROOT/pages/advanced-how-tos/releasing/tenant-release-pipelines.adoc

```diff
@@ -48,11 +48,11 @@ spec:
      resolver: git
      params:
        - name: url
-         value: "https://github.com/konflux-ci/release-service-catalog.git"
+         value: "https://github.com/github-user/pipeline-repo.git"
```

Comment:
Is this just a change to indicate that we don't have an out-of-the-box pipeline ready for tenant release examples?

If so, can we make this more explicit? Something like `https://github.com/<your-github-user>/<your-pipeline-repo>.git`

Reply:
updated

Reply:
Once https://github.com/konflux-ci/release-service/issues/589 is done, its result could go here. But for now, it doesn't make sense as written